packages/cxl-ui/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.0-alpha.7",
   "author": "CXL <leho@cxl.com>",
   "license": "SEE LICENSE IN LICENSE",
-  "main": "src/index.js",
+  "main": "src/index-storybook.js",
```

Comment:
What's happening here, how come cxl-ui main is something `...-storybook`?

Reply:
> What's happening here, how come cxl-ui main is something `...-storybook`?

I created a separate entry point for storybook.
We have two separate bundles, one for core ui & one for playbooks.
Didn't find a way to add both here, so storybook can combine them and load both. So opted to create unique entry point for storybook.

Also here are two approaches, let me know which one you like:

Old
https://github.com/conversionxl/aybolit/commits/1a57ef8
current
https://github.com/conversionxl/aybolit/commits/e6d74e2